test/unit/cli.js

```diff
@@ -122,7 +122,7 @@ test('cli errors if provider does not exist', function (t) {
 
 test('cli warns if sauce labs credentials are provided in root config', function (t) {
   const cwd = tempy.directory()
-  const expected = read('deprecated-creds.txt')
+  const expected = read('deprecated-creds.txt').toString().trim()
```

Comment:
```suggestion
  const expected = read('deprecated-creds.txt').trim()
```

Reply:
(I'm not sure what the intent is here, but toString() isn't required because it's already a string)

Reply:
@vweevers WebStorm Intellisense got confused due to additional wrapping of the read function and somehow inferred this overloads ends in Buffer, while it actually is a String, since encoding is provided. Fixed/